chain_info/Ajuna.yaml

```diff
@@ -0,0 +1,31 @@
+name: Ajuna
+network_type: mainnet
+chain:
+  type: parachain
+  parent: polkadot
+  consensus: null
+  sudo: true
+  para_id: 2051
+chainspec:
+  http_url: https://raw.githubusercontent.com/ajuna-network/Ajuna/main/resources/ajuna/ajuna-raw.json
+goals: []
+repository: https://github.com/ajuna-network/Ajuna
+specs:
+  block_time: 12
+  ed: 1000000000
+  decimals: 12
+  token: AJUN
+  ss58_format: 1328
+contacts:
+  - type: email
+    contact: hello@ajuna.io
+documentation: []
```

Comment:
Would be good to come back and add this when you guys have the documentation ready.